packages/lodestar/src/sync/regular/naive/naive.ts

```diff
@@ -154,12 +166,39 @@ export class NaiveRegularSync implements IRegularSync {
   }
 
   private getSyncPeers = async (): Promise<PeerId[]> => {
-    return this.network.getPeers().reduce( (validPeers: PeerId[], peer: PeerId) => {
-      const rep = this.reps.getFromPeerId(peer);
-      if(rep && rep.latestStatus) {
-        validPeers.push(peer);
+    if (!this.network.getPeers().includes(this.bestPeer)) {
+      // bestPeer disconnected
+      this.bestPeer = undefined;
+      await this.waitForBestPeer();
+    }
+    return [this.bestPeer];
+  };
+
+  private waitForBestPeer = async (): Promise<void> => {
+    // check 2 times per slot
+    const waitingTime = this.config.params.SECONDS_PER_SLOT * 1000 / 2;
```

Comment:
Isn't this too often? How often does other clients refresh sync status?

Maybe we should run status update as daemon somewhere else?

Reply:
I changed `waitingTime` to per slot as it's more reasonable.

I don't see the need to create a daemon for now unless we want to prepare something for the future. In that case we'll decide the waitingTime depending on different use cases for it.

For this use case, we just need this for a short period, created `syncPeersStatus()` util for it.